apps/shipments/views/shipment_overview.py

```diff
@@ -110,8 +115,14 @@ def get_queryset(self, *args, **kwargs):
                                    .values('id'))
 
         if settings.PROFILES_ENABLED:
-            # Filter by owner
-            queryset = queryset.filter(shipment__owner_id=get_owner_id(self.request))
+            queryset_filter = Q(shipment__owner_id=get_owner_id(self.request))
+            permission_link_id = self.request.query_params.get('shipment__permission_link')
```

Comment:
I don't think we are actually supporting permission links on the shipment overview page, just the shipment detail page.